test/unittests/t_uuid.cc

```diff
@@ -62,7 +62,7 @@ TEST(T_Uuid, FailRead) {
   fclose(f);
   UnlinkGuard unlink_guard(path);
   UniquePtr<Uuid> uuid(Uuid::Create(path));
-  EXPECT_TRUE(uuid == NULL);
+  EXPECT_TRUE(uuid.IsValid());
```

Comment:
True... Thanks for reviewing carefully. [Fixed](https://github.com/cvmfs/cvmfs/pull/668), also the one above was quirked. However, I wonder, because yesterday the test didn't fail, I believe...